docs/src/content/docs/guides/site-search.mdx

```diff
@@ -42,6 +42,44 @@ This text will be hidden from search.
 </div>
 ```
 
+## Adjust ranking in search results
+
+Pagefind's index ranking can be adjusted using the [`data-pagefind-weight`](https://pagefind.app/docs/weighting/#ranking-content-higher-or-lower) attribute. This can be used to [adjust a whole page](#adjust-whole-page-rank) or [individual sections of a page](#adjust-individual-section-rank).
+
+Learn more about about how the weight value impacts search results in the [Pagefind Weighting content guide](https://pagefind.app/docs/weighting/).
```

Comment:
It would be nice to explain to users when they should change the ranking, and why. The docs of pagefind only explain some technical stuff, but from the point of view of maintainer as myself, it would be nice to have some guidance on why I should use it.

Reply:
```suggestion
## Adjust ranking in search results

Pagefind's index ranking can be adjusted using the [`data-pagefind-weight`](https://pagefind.app/docs/weighting/#ranking-content-higher-or-lower) attribute. This can be used to [adjust a whole page](#adjust-whole-page-rank) or [individual sections of a page](#adjust-individual-section-rank).

For example, if there is a high-level guide that is a better entry point for readers than an error reference page with similar keywords, adjusting the weight of the guide page will cause it to be ranked higher in the Pagefind search results. 

Learn more about about how the weight value impacts search results in the [Pagefind Weighting content guide](https://pagefind.app/docs/weighting/).
```

@ematipico Thanks! Do you think this gives a good example of the usage here? Happy for any suggestions if anything else comes to mind 🥳

Reply:
We're getting somewhere I think. Now we should explain when to **increase** the page weight and when **decrease** the page weight.